DEVELOPER_SETUP.md

```diff
@@ -80,49 +80,42 @@ cp ./bin64/RAHasher /usr/bin/RAHasher
 
 #### - Install python dependencies
 
-You'll need poetry installed
+You'll need uv installed
 
-<https://python-poetry.org/docs/#installing-with-the-official-installer>
+<https://docs.astral.sh/uv/getting-started/installation/>
 
 ```sh
-pipx install poetry
+curl -LsSf https://astral.sh/uv/install.sh | sh
 ```
 
-Then create the virtual environment
+Then create the virtual environment and install the dependencies using Poetry:
 
-```sh
-# Fix disable parallel installation stuck: $> poetry config experimental.new-installer false
-# Fix Loading macOS/linux stuck: $> export PYTHON_KEYRING_BACKEND=keyring.backends.null.Keyring
-poetry sync --all-extras
-```
-
-If you are on Arch Linux or another Arch-based distro, you need to run the command as follows:
-
-```sh
-# https://bbs.archlinux.org/viewtopic.php?id=296542
-CFLAGS="-Wno-error=incompatible-pointer-types" poetry sync --all-extras
-```
+````sh
+uv venv
+source .venv/bin/activate
+uv sync --all-extras --dev
+``
```

Comment:
~~~suggestion
```sh
uv sync --all-extras
```
~~~

Reply:
I think for dev setup we should push ppl to create a venv and source it, for cleanliness